bare-metal/elastic-metal/how-to/use-private-networks.mdx

```diff
@@ -128,16 +128,16 @@ You must configure the virtual network interface on each Elastic Metal server yo
     </Message>
 2. Launch the **Server Manager** application.
 3. Click **Local Server** in the menu on the left. Then click **NIC Teaming** in the **Properties** section.
```

Comment:
These steps seem to be out of order?

Reply:
@RoRoJ This one made my brain bug, but I think it should be fine now. 
It seems there was one extra step that was just a mashup of other steps.